ui/apps/platform/src/Containers/Audit/ListeningEndpoints/ListeningEndpointsTable.tsx

```diff
@@ -44,18 +45,21 @@ function EmbeddedTable({
 
 export type ListeningEndpointsTableProps = {
     deployments: (ListDeployment & { listeningEndpoints: ProcessListeningOnPort[] })[];
+    getSortParams: GetSortParams;
 };
 
-function ListeningEndpointsTable({ deployments }: ListeningEndpointsTableProps) {
+function ListeningEndpointsTable({ deployments, getSortParams }: ListeningEndpointsTableProps) {
     const expandedRowSet = useSet<string>();
     return (
         <TableComposable variant="compact">
             <Thead noWrap>
                 <Tr>
                     <Th>{/* Header for expanded column */}</Th>
-                    <Th width={30}>Deployment</Th>
-                    <Th>Namespace</Th>
-                    <Th>Cluster</Th>
+                    <Th width={30} sort={getSortParams('Deployment')}>
+                        Deployment
+                    </Th>
+                    <Th sort={getSortParams('Namespace')}>Namespace</Th>
+                    <Th sort={getSortParams('Cluster')}>Cluster</Th>
```

Comment:
I had to do a double-take on the hook function's name, `getSortParams`, which I found actually returns an object according to PatternFly's needs.

I see that is the convention the PF docs use. (sigh)